sphinx/ext/autodoc.py

```diff
@@ -1637,13 +1637,16 @@ class AttributeDocumenter(DocstringStripSignatureMixin, ClassLevelDocumenter):
     # some non-data descriptors as methods
     priority = 10
 
-    method_types = (FunctionType, BuiltinFunctionType, MethodType)
+    @staticmethod
+    def is_function_or_method(obj):
```

Comment:
Memo: this method should be module function or inner function instead of staticmethod.
However this is not a strong opinion in this case. 

Note: this method is not a API, so we may change the name or move to the function at some timing.